lib/operations/mongo_client_ops.js

```diff
@@ -416,6 +417,43 @@ function createTopology(mongoClient, topologyType, options, callback) {
     }
 
     assignTopology(mongoClient, newTopology);
+    if (options.autoEncryption == null) {
+      callback(null, newTopology);
+      return;
+    }
+
+    // setup for client side encryption
+    let AutoEncrypter;
+    try {
+      AutoEncrypter = require('mongodb-client-encryption').AutoEncrypter;
+    } catch (err) {
+      callback(
+        new MongoError(
+          'Auto-encryption requested, but the module is not installed. Please add `mongodb-client-encryption` as a dependency of your project'
+        )
+      );
+
+      return;
+    }
+
+    const MongoClient = loadClient();
+    const connectionString =
+      os.platform() === 'win32'
+        ? 'mongodb://localhost:27020/?serverSelectionTimeoutMS=1000'
```

Comment:
q: should this be configurable? Is it required that the `mongocryptd` be on `localhost:27020`?

Reply:
I think it should be configurable in follow-on work, created [NODE-2012](https://jira.mongodb.org/browse/NODE-2012) to track this